lib/active_record/sql_analyzer/configuration.rb

```diff
@@ -150,11 +150,12 @@ def setup_defaults
           Redactor.new(/\n/, " "),
           Redactor.new(/\s+/, " "),
           Redactor.new(/(\s|\b|`)(=|!=|>=|>|<=|<) ?(BINARY )?-?\d+(\.\d+)?/, " = [REDACTED]"),
-          Redactor.new(/(\s|\b|`)(=|!=|>=|>|<=|<) ?(BINARY )?x?'.*?[^\\]'/, " = '[REDACTED]'"),
+          Redactor.new(/(\s|\b|`)(=|!=|>=|>|<=|<) ?(BINARY )?x?'.*[^\\]'/, " = '[REDACTED]'"), # Special case, to deal with ' in the values being set
+	  Redactor.new(/(\s|\b|`)(=|!=|>=|>|<=|<) ?(BINARY )?x?'.*?[^\\]'/, " = '[REDACTED]'"), # More general case, deals with empty values
           Redactor.new(/VALUES \(.+\)$/, "VALUES ([REDACTED])"),
           Redactor.new(/IN \([^)]+\)/, "IN ([REDACTED])"),
           Redactor.new(/BETWEEN '[^']*' AND '[^']*'/, "BETWEEN '[REDACTED]' AND '[REDACTED]'"),
-          Redactor.new(/LIKE '[^'\\]*(?:\\.[^'\\]*)*'/, "LIKE '[REDACTED]'"),
+          Redactor.new(/(LIKE|like) '[^'\\]*(?:\\.[^'\\]*)*'/, "LIKE '[REDACTED]'"), # Special case, sub-queries may not be upcased
```

Comment:
technically this should probably be done for all of the keywords, because in active record you can use lowercase keywords if you construct a query by hand